temoa-uiuc/data_files/11_uiuc.sql

```diff
@@ -1314,6 +1314,51 @@ INSERT INTO "CostFixed" VALUES('uiuc', 2028, 'GEOT', 2020, 0.0469, 'M$/MWth', 'c
 INSERT INTO "CostFixed" VALUES('uiuc', 2029, 'GEOT', 2020, 0.0469, 'M$/MWth', 'cost of O&M of the plant');
 INSERT INTO "CostFixed" VALUES('uiuc', 2030, 'GEOT', 2020, 0.0469, 'M$/MWth', 'cost of O&M of the plant');
 --
+<<<<<<< HEAD
```

Comment:
One reason these scenarios might not be working is because you have an unresolved merge conflict.